### ISSUE TITLE
Complex job

### ISSUE THREAD

**Issue by robertstefan** [quartznet/quartznet#721]
Happy new year, first of all !

I am in a bit of a situation here guys as I did not find anything to work .. 

I'm using Quartz as a Windows service on a Windows Server 2016, w/ MySQL as job-store and I need to register a job.
Registering simple jobs is not a problem, Quartz does the job as it should, but I have the following scenario:
- a job that requires a rather big collection of dlls.

For the moment, it's the only job of it's kind, and I was wondering if it is possible to deploy the job in the form of 
```
.
+-- Quartz.Server.exe
+-- *.dll
+-- myjob.dll
+-- myjob.libs
|   +-- *.dll
|   +-- *.config
```
where _myjobs.libs_ would obviously contain all the dlls that the job needs to run.

After some research, I found a solution to use in app.config the following:
```xml
<runtime>
    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
      <probing privatePath="myjob.libs" />
 <!-- ... -->
    </assemblyBinding>
</runtime>
```

It does find my job upon startup, it registers the job, but afterwards nothing is happening. (Maybe it has something with the logging framework used by the job dll? - it's using NLog)

How exactly should I tackle this scenario?
Any idea is welcomed.

**Comment by lahma** [comment]
Can you enable more logging? Quartz (3.x) should automatically pick up your logging framework and output there. Logging should reveal problems quite well when you get it working, after starting the scheduler there should be all kinds of output.

**Comment by robertstefan** [comment]
Sorry for the late reply. 
So, I monitored the log file generated by my job and Quartz wrote there everything and found the following issues:

1. Had a lot of issues with _bindingRedirect_
And,
2. after finally managing to make it work with all the bindingRedirects, the job never got fired, nothing got written in the logs.

In the end, I approached the issue differently by splitting the job in a 2 parts: 
- a simple job file that will publish in a message queue;
- a win service that will consume that queue and will execute the 'job'.

Maybe there was still some things that I missed if there were no logs.
